deploy/kubernetes/deployment.yaml

```diff
@@ -34,8 +36,8 @@ spec:
         - name: csi-provisioner
           image: docker.io/k8scsi/csi-provisioner
```

Comment:
How should we keep this image up to date whenever we branch/release?

Reply:
Fixed (using canary on quay.io) and fixed storage class in examples too (it used csi-flex driver from some reason).

Reply:
We'll have to remember to update this when we branch